src/loaders/compute_tools/busco/busco.py

```diff
@@ -0,0 +1,81 @@
+"""
+Run BUSCO tool on a set of fna files.
+
+This tool serves a distinct purpose separate from collection tools; instead, it is suited for CDM work.
+Therefore, the parser program is not compatible with data generated by this tool.
+
+"""
+import os
+import time
+from pathlib import Path
+
+from src.loaders.common.loader_common_names import TOOL_METADATA
+from src.loaders.compute_tools.tool_common import ToolRunner, run_command, create_tool_metadata
+from src.loaders.compute_tools.tool_version import extract_latest_reference_db_version
+
+
+def _run_busco_single(
+        tool_safe_data_id: str,
+        data_id: str,
+        source_file: Path,
+        output_dir: Path,
+        threads_per_tool_run: int,
+        debug: bool) -> None:
+    start = time.time()
+    print(f'Start executing BUSCO for {data_id}')
+
+    metadata_file = output_dir / TOOL_METADATA
+    if metadata_file.exists():
+        print(f"Skipping {source_file} as it has already been processed.")
+        return
+
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    version_file = os.path.join(current_dir, 'versions.yaml')
+    ref_db_version = extract_latest_reference_db_version(version_file)
+
+    command = [
+        'busco',
+        '-i', str(source_file),
+        '-o', data_id,
+        '--out_path', str(output_dir),
+        '--datasets_version', ref_db_version,
+        '--download_path', '/reference_data',
+        '-c', str(threads_per_tool_run),
+        '--auto-lineage-prok',
+        '-m', 'genome',
+        '-f',
+        '--augustus',
```

Comment:
brief inline comments as to what these args do would be helpful

Reply:
added comments.